package/network/config/qos-scripts/files/usr/lib/qos/generate.sh

```diff
@@ -2,11 +2,10 @@
 [ -e /lib/functions.sh ] && . /lib/functions.sh || . ./functions.sh
 [ -x /sbin/modprobe ] && {
 	insmod="modprobe"
-	rmmod="$insmod -r"
 } || {
 	insmod="insmod"
-	rmmod="rmmod"
 }
+rmmod="rmmod"
```

Comment:
Hi, hope you don't mind... Considering we are dropping this, can't we just hardcode and drop the variable?